pkgs/development/python-modules/bayesian-optimization/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, scikitlearn
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "bayesian-optimization";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "fmfn";
+    repo = "BayesianOptimization";
+    rev = "v${version}";
+    sha256 = "07sqymg6k5512k7wq4kbp7rsrkb4g90n0ck1f0b9s6glyfpcy4pq";
+  };
+
+  propagatedBuildInputs = [
+    scikitlearn
+    scipy
+  ];
+  
+  checkInputs = [ pytest ];
+  checkPhase = ''
+    pytest \
+      tests/test_acceptance.py \
+      tests/test_observer.py \
+      tests/test_util.py \
+      tests/test_bayesian_optimization.py \
+      tests/test_queue.py \
+      tests/test_target_space.py
```

Comment:
since you're not filtering out any of the tests under the `tests` directory, you can just do:
```
checkPhase = ''
    pytest tests
'';
```

Reply:
otherwise LGTM

Reply:
Fixed. Thanks.